src/python/grpcio/build.py

```diff
@@ -0,0 +1,118 @@
+# Copyright 2016, Google Inc.
```

Comment:
Are you sure you read that article all the way through? Command-files are an opt-in feature only stylistically recommended. Further, `gcc` accepts command files in a different manner than the toy example provided by Microsoft.